documentsManager/admin.py

```diff
@@ -1,7 +1,30 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import ugettext_lazy as _
 
-from .models import FileDoc, FileType, Event
+from .models import FileDoc, FileType, Event, User
```

Comment:
```suggestion
from .models import FileDoc, FileType, Event, User
``` 
Pls indent when the imports is too long

Reply:
Ok.